### PR TITLE
add obis-observer command GetShortName

### PR DESCRIPTION
**Pull request by DeadByElpy**

### CODE REVIEW

File: docs/obis-observer/commands/GetShortName.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+| ----- | ------ | ------------------------- |
+| `1`   | `byte` | command id = `0x01`       |
+| `1`   | `byte` | [request id](#request-id) |
+| `3-7` | `byte` | [obis](#obis)             |
+
+### Parameters
+
+#### **request id**
+
+It's a unique number for pair device and GetShortName correction request.
```
Comment (DarkPark) [inline docs/obis-observer/commands/GetShortName.md:20]:
```suggestion
It's a unique number to link request with response.
```

File: docs/obis-observer/commands/GetShortName.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+Then increment this value for each following request.
```
Comment (DarkPark) [inline docs/obis-observer/commands/GetShortName.md:21]: It should be incremented for each request.

File: docs/obis-observer/commands/GetShortName.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+| `3-7` | `byte` | [obis](#obis)                |
+| `1`   | `byte` | short name `1`               |
+| ...   | ...    | ...                          |
+| `1`   | `byte` | short name `N`               |
+
+### Parameters
+
+#### **request id**
+
+It's a unique number for pair device and GetShortName correction request.
```
Comment (DarkPark) [inline docs/obis-observer/commands/GetShortName.md:61]: also would be nice to correct

File: docs/obis-observer/types.md [hunk 4]
```diff
@@ -0,0 +1,211 @@
+# Special data types
+
+Commands use many specific data types packed to bytes.
+This document describes these types and structuring principles.
+
+
+## OBIS
+
+It's a OBIS code packed in `3-7` bytes.
```
Comment (DarkPark) [inline docs/obis-observer/types.md:9]:
```suggestion
It's an OBIS code packed in `3-7` bytes.
```

File: docs/obis-observer/types.md [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+`4` bytes are necessary for this.
+
+The group values:
+
+| group | value | hex    | bits       |
+| ----- | ----- | ------ | ---------- |
+| `A`   | `1`   | `0x01` | `0b1`      |
+| `B`   | `0`   |        |            |
+| `C`   | `11`  | `0x0b` | `0b1011`   |
+| `D`   | `1`   | `0x23` | `0b100011` |
```
Comment (DarkPark) [inline docs/obis-observer/types.md:75]: wrong value?

File: docs/obis-observer/types.md [hunk 6]
```diff
@@ -0,0 +1,211 @@
...
+            <td><code>0</code></td>
+            <td><code>1</code></td>
+            <td><code>0</code></td>
+            <td><code>1</code></td>
+            <td><code>0</code></td>
+            <td><code>0</code></td>
+            <td><code>1</code></td>
+        </tr>
+        <tr>
+            <td colspan="8" align="center"><code>0</code></td>
```
Comment (DarkPark) [inline docs/obis-observer/types.md:196]: maybe it's better not to join cells here?

**Review by DarkPark — CHANGES_REQUESTED** [review]

File: docs/obis-observer/types.md (hunk 6 above)
Comment (DeadByElpy) [reply docs/obis-observer/types.md]: reworked as 8 cells of bits

File: docs/obis-observer/types.md (hunk 5 above)
Comment (DeadByElpy) [reply docs/obis-observer/types.md]: fixed

File: docs/obis-observer/types.md (hunk 4 above)
Comment (DeadByElpy) [reply docs/obis-observer/types.md]: fixed

File: docs/obis-observer/commands/GetShortName.md (hunk 3 above)
Comment (DeadByElpy) [reply docs/obis-observer/commands/GetShortName.md]: fixed

File: docs/obis-observer/commands/GetShortName.md (hunk 2 above)
Comment (DeadByElpy) [reply docs/obis-observer/commands/GetShortName.md]: description for request id changed

**Pull request merged**